src/cf-typography/usage.md

```diff
@@ -57,18 +121,20 @@
 
 ### Large pull quote
 
-<aside class="pull-quote pull-quote__large">
-    <div class="pull-quote_body">
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-        Cum corrupti tempora nam nihil qui mollitia consectetur
-        corporis nemo culpa dolorum!
-    </div>
-    <footer>
-        <cite class="pull-quote_citation">
-            - Author Name
-        </cite>
-    <footer>
-</aside>
+<div class="block">
+    <aside class="pull-quote pull-quote__large">
+        <div class="pull-quote_body">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Cum corrupti tempora nam nihil qui mollitia consectetur
+            corporis nemo culpa dolorum!
+        </div>
+        <footer>
+            <cite class="pull-quote_citation">
+                - Author Name
+            </cite>
+        </footer>
+    </aside>
+</div>
```

Comment:
Hmm... sure we want to wrap these in `.block`? That'll add a lot of default margin that I don't think we want these to have, do we?